alphadia/planning.py

```diff
@@ -158,14 +157,22 @@ def spectral_library(self, spectral_library: SpecLibFlat) -> None:
         self._spectral_library = spectral_library
 
     def log_environment(self):
-        logger.progress("=================== Environment ===================")
+        logger.progress("================ AlphaX Environment ===============")
         logger.progress(f"{'alphatims':<15} : {alphatims.__version__:}")
         logger.progress(f"{'alpharaw':<15} : {alpharaw.__version__}")
         logger.progress(f"{'alphabase':<15} : {alphabase.__version__}")
         logger.progress(f"{'alphapeptdeep':<15} : {peptdeep.__version__}")
         logger.progress(f"{'directlfq':<15} : {directlfq.__version__}")
         logger.progress("===================================================")
 
+        logger.progress("================= Pip Environment =================")
+        pip_env = [
+            f"{dist.metadata['Name']}=={dist.version}"
```

Comment:
I thought about using debug Level logging. Bit I guess it doesnt hurt to have this by default.

Reply:
you will never need this until you need it, so I would leave it on by default

what about the `progress` level anyway? could this not be INFO?

Reply:
The idea is that progress has the important stuff and is colored green so you can find it faster when checking output